src/main/java/shizuka/Shizuka.java

```diff
@@ -0,0 +1,102 @@
+package shizuka;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Shizuka {
+    static final String FILE_PATH = "shizuka.txt";
+
+    public static String[] parseCommand(String args) {
+        return args.split(" ", 2);
+    }
+
+    public static int parseNumber(String args) {
+        int endIndex = args.indexOf(' ') + 1;
+        return Integer.parseInt(args.substring(endIndex));
+    }
+
+    public static void main(String[] args) {
+        Printer.intro();
+        Scanner in = new Scanner(System.in);
+        String line, lineTrimmed;
+        TodoList list0 = new TodoList();
+
+        do {
+            line = in.nextLine();
+            lineTrimmed = line.trim();
+            String[] command = parseCommand(lineTrimmed);
+            int taskNum;
+            switch (command[0]) {
```

Comment:
Consider having a `Command` class to deal with the command code, to apply more OOP principles and reduce repetitive code.